apiserver/leadership.go

```diff
@@ -89,3 +90,26 @@ func (m leadershipPinner) UnpinLeadership(applicationName string, entity string)
 func (m leadershipPinner) PinnedLeadership() map[string][]string {
 	return m.pinner.Pinned()
 }
+
+// leadershipReader implements leadership.Reader by wrapping a lease.Reader.
+type leadershipReader struct {
```

Comment:
should we call this raftLeadershipReader just to be more obvious about the distinction?

Reply:
I actually had it called that initially :)
Because it technically only deals with indirections, I thought the legacy vs vanilla was sufficient.